kubernetes/argocd/stacks/chicago-edt/main.yml

```diff
@@ -0,0 +1,208 @@
+---
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: chicago-edt-ingress
+  namespace: chicago-edt
+  annotations:
+    external-dns.alpha.kubernetes.io/target: app.acmuic.org
+spec:
+  entryPoints:
+    - websecure
+  routes:
+    - kind: Rule
+      match: "Host(`edtvm.acmuic.org`)"
+      services:
+        - kind: Service
+          name: chicago-edt-svc
+          namespace: chicago-edt
+          passHostHeader: true
+          port: http
+          responseForwarding:
+            flushInterval: 1ms
+          scheme: http
+          strategy: RoundRobin
+          weight: 10
+  tls:
+    secretName: chicago-edt-tls
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: chicago-edt-tls
+spec:
+  dnsNames:
+    - edtvm.acmuic.org
+  secretName: chicago-edt-tls
+  issuerRef:
+    kind: ClusterIssuer
+    name: acmuic-self-ca
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: wordpress-pvc
+  namespace: chicago-edt
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: nfs
+  resources:
+    requests:
+      storage: 20Gi
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: database-pvc
+  namespace: chicago-edt
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: nfs
+  resources:
+    requests:
+      storage: 2Gi
```

Comment:
I would probably give them a bit more. Maybe 10Gi? Their current usage is already about ~810MB.